### PR TITLE
[Pages] API with Workers tutorial revision

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: products/pages/src/content/tutorials/build-an-api-with-workers/index.md [hunk 1]
```diff
@@ -6,19 +6,19 @@ pcx-content-type: tutorial
...
 
-This tutorial contains two, separate applications: the backend, a [serverless](https://www.cloudflare.com/learning/serverless/what-is-serverless/) API deployed on Cloudflare Workers, and the frontend, built with React and deployed using Cloudflare Pages.
+This tutorial contains two, separate applications: the backend, a [serverless](https://www.cloudflare.com/learning/serverless/what-is-serverless/) API deployed on Cloudflare Workers, and the front end, built with React and deployed using Cloudflare Pages.
 
-If you are interested in a more comprehensive approach to building applications like this, you may benefit from moving from a serverless API for your content to a headless CMS — see our [headless CMS tutorial] to learn more.
+If you are interested in a more comprehensive approach to building applications like this, you may benefit from moving from a serverless API for your content to a headless CMS — refer to the [headless CMS tutorial] to learn more.
```
Comment (kodster28) [inline products/pages/src/content/tutorials/build-an-api-with-workers/index.md:13]: Missing link for headless CMS tutorial?

File: products/pages/src/content/tutorials/build-an-api-with-workers/index.md [hunk 2]
```diff
@@ -420,15 +422,15 @@ route = "*myblog.com/api*"
...
 ## Conclusion
 
-In this tutorial, you built a full blog application by combining a frontend deployed with Cloudflare Pages, and a serverless API built with Cloudflare Workers. You can find the source code for both codebases on GitHub:
+In this tutorial, you built a full blog application by combining a front end deployed with Cloudflare Pages, and a serverless API built with Cloudflare Workers. You can find the source code for both codebases on GitHub:
 
 - Blog frontend: https://github.com/signalnerve/blog-frontend
 - Serverless API: https://github.com/signalnerve/serverless-api
 
-If you enjoyed this tutorial, check out our [headless CMS tutorial].
+If you enjoyed this tutorial, refer to the [headless CMS tutorial] to learn how to build a blog using Nuxt.js and Sanity.io.
```
Comment (kodster28) [inline products/pages/src/content/tutorials/build-an-api-with-workers/index.md:434]: Why wouldn't this just be an inline link?

**Review by kodster28 — APPROVED** [review]

File: products/pages/src/content/tutorials/build-an-api-with-workers/index.md (hunk 2 above)
Comment (kodster28) [reply products/pages/src/content/tutorials/build-an-api-with-workers/index.md:434]: Just ignore me here
download

File: products/pages/src/content/tutorials/build-an-api-with-workers/index.md [hunk 3]
```diff
@@ -395,21 +395,23 @@ export default Post;
 
 ### Publishing with Cloudflare Pages
 
-Publishing your project with Cloudflare Pages is an easy, two-step process: first, push your project to GitHub, and then in the Cloudflare Pages UI, set up a new project based on that GitHub repository. Pages will deploy a new version of your site each time you publish, and will even set up preview deployments whenever you open a new pull request.
+Publishing your project with Cloudflare Pages is an easy, two-step process: first, push your project to GitHub, and then in the Cloudflare Pages dashboard, set up a new project based on that GitHub repository. 
 
-To push your project to GitHub, [create a new repo](https://repo.new), and follow the instructions to push your local Git repository to GitHub.
+Pages will deploy a new version of your site each time you publish, and will even set up preview deployments whenever you open a new pull request.
```
Comment (lukeed) [inline products/pages/src/content/tutorials/build-an-api-with-workers/index.md]:
```suggestion
Pages will deploy a new version of your site each time you publish and will set up preview deployments whenever you open a new pull request.
```

File: products/pages/src/content/tutorials/build-an-api-with-workers/index.md [hunk 4]
```diff
@@ -395,21 +395,23 @@ export default Post;
...
 
-Once you have pushed your project to GitHub, go to the Pages UI, and create a new project. When asked for your project's build configuration, just choose "React" -- Pages will set the correct fields for you automatically.
+To push your project to GitHub, [create a new repository](https://repo.new), and follow the instructions to push your local Git repository to GitHub.
+
+Deploy your site to Pages by logging into the [Cloudflare dashboard](https://dash.cloudflare.com/) > **Account Home** > **Pages** and selecting **Create a project**. Select the new GitHub repository that you created and, in the **Set up builds and deployments** section, choose **React** -- Pages will set the correct fields for you automatically.
```
Comment (lukeed) [inline products/pages/src/content/tutorials/build-an-api-with-workers/index.md]:
```suggestion
Deploy your site to Pages by logging into the [Cloudflare dashboard](https://dash.cloudflare.com/) > **Account Home** > **Pages** and selecting **Create a project**. Select the new GitHub repository that you created and, in the **Set up builds and deployments** section, choose **React** -- Pages will automatically apply the correct build settings for you.
```

File: products/pages/src/content/tutorials/build-an-api-with-workers/index.md [hunk 5]
```diff
@@ -420,15 +422,15 @@ route = "*myblog.com/api*"
 zone_id = "$zoneId"
 ```
 
-Now, when you run `wrangler publish`, your API will be published and served on `myblog.com`, but _only_ when you visit a route matching `/api*` -- this means that your basic routes (`/`, `/posts/:id`, etc.) will be sent to your frontend Pages application, but any API routes will be intercepted and handled by your Workers application.
+Now, when you run `wrangler publish`, your API will be published and served on `myblog.com`, but only when you visit a route matching `/api*` -- this means that your basic routes (`/`, `/posts/:id`, etc.) will be sent to your front-end Pages application, but any API routes will be intercepted and handled by your Workers application.
 
 ## Conclusion
 
-In this tutorial, you built a full blog application by combining a frontend deployed with Cloudflare Pages, and a serverless API built with Cloudflare Workers. You can find the source code for both codebases on GitHub:
+In this tutorial, you built a full blog application by combining a front end deployed with Cloudflare Pages, and a serverless API built with Cloudflare Workers. You can find the source code for both codebases on GitHub:
```
Comment (lukeed) [inline products/pages/src/content/tutorials/build-an-api-with-workers/index.md:429]: This article now flip-flops a ton between `front-end` and `front end` – neither of which are actually the common form in the programming community at large.

Comment (deadlypants1973) [reply products/pages/src/content/tutorials/build-an-api-with-workers/index.md:429]: @lukeed 'front end' noun, 'front-end' compound adjective reasoning

Comment (lukeed) [reply products/pages/src/content/tutorials/build-an-api-with-workers/index.md:429]: Understood, but frontend is effectively a proper noun. As a term, it's always referring to the same thing 🤷

**Pull request merged**